sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/TypeOfActivity.java

```diff
@@ -0,0 +1,24 @@
+package org.sormas.e2etests.enums;
+
+public enum TypeOfActivity {
```

Comment:
we already have:
**public enum ActivityTypes**
Please rename new created to reflect the need, and update also the existing one